### ISSUE TITLE
TypeError: Cannot read property '_id' of undefined When I try to add a product

### ISSUE THREAD

**Issue by HackIvan** [jaewonhimnae/react-shop-app#6]
55 | }
  56 | 
  57 | const variables = {
> 58 |     writer: props.user.userData._id,
     | ^  59 |     title: TitleValue,
  60 |     description: DescriptionValue,
  61 |     price: PriceValue,

I've created a mongo db account and I've created the dev.js file adding the credentials to in for accessing the database

**Comment by vishchi** [comment]
Yeah one thing, he mistakenly kept those <> around password and dbname inside the dev.js(config) file. You are supposed to remove those. It should look like: module.exports = {
    mongoURI:'mongodb+srv://bla_user:bla_pass@cluster0.skpel.mongodb.net/test?retryWrites=true&w=majority'
}
